package.json

```diff
@@ -44,7 +44,7 @@
     "test:e2e:chrome:rpc": "SELENIUM_BROWSER=chrome node test/e2e/run-all.js --rpc",
     "test:e2e:firefox": "SELENIUM_BROWSER=firefox node test/e2e/run-all.js",
     "test:e2e:firefox:snaps": "SELENIUM_BROWSER=firefox node test/e2e/run-all.js --snaps",
-    "test:e2e:single": "node test/e2e/run-e2e-test.js",
+    "test:e2e:single": "UPDATE_SNAPSHOTS=true node test/e2e/run-e2e-test.js",
```

Comment:
Remove `UPDATE_SNAPSHOTS`. I don't think that we want to have it as a default when running an e2e test.